### PR TITLE
Jrosinsk/wip make etcd load balancer optional poc

### LINKED ISSUE #32: Add configuration support to allow disabling the etcd load balancer
Add a parameter to support the enable/disable of using an etcd load balancer.

etcdLBEnabled = "false"

### PR DESCRIPTION
**Pull request by jrosinsk**
Fixes #32 This transaction adds a flag to disable the etcd load balancer if desired. The default behavior is to use the etcd load balancer.  To enable this feature, etcdLBEnabled="false" can be set in the tfvars file. 

Successfully ran the sonobuoy e2e cluster tests on a cluster with this flag enabled.

### CODE REVIEW

**Comment by j0nr33v3** [comment]
Can you briefly describe what happens when etcdLBEnabled="true" i.e. etcd is run on the master nodes? (each master node uses a local etcd instance?)

**Comment by jrosinsk** [comment]
Hi Jon,

Sure, etcdLBEnabled only controls the existence of the the etcd load
balancer.  So, when it is "true" the configuration is exactly as it was
before this flag was introduced; "true" is also the default setting.

Adding support for running etcd instances and master instances on the same
node is the next step.

Regards,
-Joe

**Comment by jrosinsk** [comment]
Updated the README for better clarity on the options for this flag.

**Comment by jlamillan** [comment]
Should probably rename your branch to remove the WIP.

File: instances/etcd/cloud_init/bootstrap.template.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ docker run -d \
 	-discovery ${etcd_discovery_url}
 
 # Download etcdctl client etcd_ver
-curl -L --retry 3 https://github.com/coreos/etcd/releases/download/${etcd_ver}/etcd-${etcd_ver}-linux-amd64.tar.gz -o /tmp/etcd-${etcd_ver}-linux-amd64.tar.gz
+curl -L --retry 10 https://github.com/coreos/etcd/releases/download/${etcd_ver}/etcd-${etcd_ver}-linux-amd64.tar.gz -o /tmp/etcd-${etcd_ver}-linux-amd64.tar.gz
```
[the remark this answers is not in the document]
Comment (jrosinsk) [reply instances/etcd/cloud_init/bootstrap.template.sh]: I'll make it 5.  Helps to insure installation of etcdctl and in turn makes the tests more likely to pass out of the box.

**Comment by jrosinsk** [comment]
renaming the branch closes the pull request. so to keep the history of this merge, keeping the branch name as is will change it before pr next time.

**Pull request merged**